src/test/java/org/wildfly/test/integration/vdx/standalone/InvalidCharactersTestCase.java

```diff
@@ -0,0 +1,156 @@
+                    spaceInXmlDeclarationStandaloneXmlPrintWriter.println(" " + line);
+                }
+
+                if (line.contains("<subsystem xmlns=\"urn:jboss:domain:webservices")) {
+                    inWSSubsystem = true;
+
+                } else if (!inWSSubsystem) {
+                    missingStartOfElementStandaloneXmlPrintWriter.println(line);
+                    missingLastQuoteStandaloneXmlPrintWriter.println(line);
+
+                } else if (line.contains("</subsystem>")) {
+                    missingStartOfElementStandaloneXmlPrintWriter.println(missingStartOfElement);
+                    missingLastQuoteStandaloneXmlPrintWriter.println(missingLastQuote);
+                    inWSSubsystem = false;
+                }
+            }
+        }
+    }
+
+    /*
+     * Add space before xml declaration - <space><?xml version='1.0' encoding='UTF-8'?>
+     */
+    @Test
+    @ServerConfig(configuration = "standalone-spaceInXmlDeclaration.xml")
+    public void spaceInXmlDeclaration()throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, " ^^^^ Illegal processing instruction target (\"xml\"); xml (case insensitive)");
+        assertContains(errorLog, "| >    at [row,col {unknown-source}]: [1,6]");
+    }
+
+    /*
+     * There is bad character in xml (for example < is missing)
+     */
+    @Test
+    @ServerConfig(configuration = "standalone-missingStartOfElement.xml")
+    public void missingStartOfElement()throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, " modify-wsdl-address>true</modify-wsdl-address>");
+        assertContains(errorLog, "^^^^ Received non-all-whitespace CHARACTERS or CDATA event in nextTag()");
+
+    }
+
+    /*
+     * There is missing last quote " in <element name="foobar>
+     */
+    @Test
+    @ServerConfig(configuration = "standalone-missingLastQuote.xml")
+    public void missingLastQuoteStandaloneXml()throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, "<endpoint-config name=\"Standard-Endpoint-Config/>");
+        assertContains(errorLog, "^^^^ Unexpected character '<' (code 60) in attribute value");
+    }
+
+    /*
+     * Element with national characters - e.g. <řebříček>obecný</řebříček>
+     */
+    @Test
+    @ServerConfig()
+    public void elementWithNationalCharacters()throws Exception {
+        container().tryStartAndWaitForFail(
+                (OfflineCommand) ctx -> ctx.client.apply(GroovyXmlTransform.of(DoNothing.class, "AddElement.groovy")
+                        .subtree("path", Subtree.subsystem("webservices")).parameter("elementXml", elementWithNationalCharacters)
```

Comment:
elementWithNationalCharacters field is used only once in the test case and value can be placed directly to offline command. It's easier to find.